sql/core/src/main/scala/org/apache/spark/sql/execution/ui/SQLAppStatusStore.scala

```diff
@@ -143,3 +144,13 @@ case class SQLPlanMetric(
     name: String,
     accumulatorId: Long,
     metricType: String)
+
+class SQLAppStatusListenerData(
```

Comment:
maybe better to place it to SQLAppStatusListener as other KVStore entities are there.